lib/ansible/modules/net_tools/dnsmadeeasy.py

```diff
@@ -562,7 +560,7 @@ def main():
             record_value=dict(required=False),
             record_ttl=dict(required=False, default=1800, type='int'),
             monitor=dict(default='no', type='bool'),
-            systemDescription=dict(default=''),
+            systemDescription=dict(),
```

Comment:
One question: why is a default value of `''` not OK for `systemDescription`? Does the API require a non-empty string?